services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list,
+                 username: str,
+                 date: datetime = None) -> list[Ticket]:

```

Comment:
The return type annotation for `create_order` should specify `list[Ticket]` to indicate that the function returns a list of `Ticket` objects.